app/helpers/ui_constants.rb

```diff
@@ -474,7 +475,8 @@ module UiConstants
     "enterprise"            => _("The Enterprise"),
     "ext_management_system" => _("Selected %{tables}") % {:tables => ui_lookup(:tables => "ext_management_systems")},
     "ems_cluster"           => _("Selected %{tables}") % {:tables => ui_lookup(:tables => "ems_cluster")},
-    "vm-tags"               => _("Tagged %{tables}") % {:tables => ui_lookup(:tables => "vm")}
+    "vm-tags"               => _("Tagged %{tables}") % {:tables => ui_lookup(:tables => "vm")},
+    "tenant"		            => _("Tenant")
```

Comment:
@jrafanie  I don't know why is happen that...In my computer (see image) it's fine and if you try to edit the code via github that appear align. <img width="896" alt="captura de pantalla 2016-03-17 a las 18 20 55" src="https://cloud.githubusercontent.com/assets/3019213/13854706/147aba8e-ec6d-11e5-8bf1-83b09f1fee65.png">